### ISSUE TITLE
EaterEvals link is broken

### ISSUE THREAD

**Issue by ChaseC99** [icssc/AntAlmanac#174]
User reported:
> Clicking on a professor's name, then clicking on the button for their EaterEvals no longer directs us to the instructor's evals. It seems like they reorganized the structure of EaterEvals and I don't think it's possible to look up instructors by name anymore. You should probably remove that button.
>
> I mainly just wanted to report it; there's no need to get back to me with updates because I don't think it'll be possible to restore that feature. When I look at a specific course feedback for a professor, the URL gives away nothing that would personally identify that professor. For example, ICS 33 has three evaluations listed for Professor Pattis, but the only differences in the URLs is a four-digit number (in the case below, it's 3510)
>
> https://evaluations.eee.uci.edu/eaterevals/view/3510?return=%2Featerevals%2Fbrowse%2Fcourse%2FI%252526C%252520SCI%2F33%2FINTERMEDIATE%252520PRGRMG
>
> For his other two evals, 3510 changes to 7641 and 7642. Everything else in the URL stays the same.
>
> Besides ICS 33, I also did that check for CS 161, and it's the same situation.

EaterEvals changed up their url structure, so our previous link that used last names no longer works: https://evaluations.eee.uci.edu/eaterevals#WONG-MA

Now, it is by their by their UCI net id:
https://evaluations.eee.uci.edu/eaterevals/browse/instructor/jwongma

The PeterPortal API has internal logic that maps a webreg name to a uci net id: https://github.com/icssc-projects/peterportal-public-api/blob/master/graphql/schema.js#L181-L223
Maybe we can expose this or copy the logic?

**Comment by devsdevsdevs** [comment]
> The PeterPortal API has internal logic that maps a webreg name to a uci net id: https://github.com/icssc-projects/peterportal-public-api/blob/master/graphql/schema.js#L181-L223
> Maybe we can expose this or copy the logic?

I think it would be redundant to copy the logic so maybe we can have an endpoint to get this?

**Comment by EricPedley** [comment]
I don't think we use eater evals on our site anymore, just RMP. Is this issue safe to close? @ChaseC99

**Comment by ChaseC99** [comment]
Yeah since we don't have a map of websoc professor name to UCINetId, it'd probably be too hard to do EaterEvals.

Feel free to close it!

**Comment by ChaseC99** [comment]
Bummer cuz this would be a cool feature :(

**Comment by EricPedley** [comment]
I'll put it in the icebox in case any eater evals enthusiasts want to work on this.

**Comment by KevinWu098** [comment]
EaterEvals is officially dead 

<img width="1170" height="2532" alt="Image" src="https://github.com/user-attachments/assets/72014294-d90c-4ebb-82ac-77daf4ae6521" />
